generator/src/test/java/com/scottlogic/deg/generator/cucumber/ShorterThan.feature

```diff
@@ -162,17 +139,30 @@ Scenario: Running a 'shorterThan' request alongside a non-contradicting matching
        | null   |
        | "☠☠" |
 
-Scenario: Running a 'shorterThan' request alongside a non-contradicting matchingRegex constraint should be successful
-       And foo is shorter than 1
+@ignore
+Scenario: Running a 'shorterThan' request alongside a contradicting matchingRegex constraint should produce null
+       And foo is shorter than 2
        And foo is matching regex /[💾]{2}/
-     Then I am presented with an error message
-       And no data is created
+     Then the following data should be generated:
+       | foo    |
+       | null   |
 
-Scenario: Running a 'shorterThan' request alongside a contradicting containingRegex constraint should fail with an error message
-       And foo is shorter than 1
+@ignore
+Scenario: Running a 'shorterThan' request alongside a non-contradicting containingRegex constraint should be successful
+       And foo is shorter than 3
        And foo is containing regex /[🍩]{2}/
-     Then I am presented with an error message
-       And no data is created
+     Then the following data should be included in what is generated:
+       | foo    |
+       | null   |
+       | "🍩🍩" |
+
+@ignore
```

Comment:
Can we raise a bug on this or link to an existing?

Reply:
This is being addressed in ticket #501